### DISCUSSION TITLE
Questions about saving personalized configurations

### DISCUSSION THREAD

**Discussion by zy1075984** [AvaloniaUI/Avalonia#11915] [Q&A] [answered]
I want to develop web and desktop general-purpose applications in RC11. How do I save the profile information in the application? Is there a recommended technical path? Thank you!

**Comment by zy1075984** [comment, marked as answer]
I'm thinking about whether to use sqlite or local File or localStorage technology, what should I do that is standard and elegant

**Comment by timunie** [comment]
Depends on your needs and the OS you use. If you want it to be stored in a file, you may need to write a service on each platform. On Desktop I use a json file via: https://docs.avaloniaui.net/guides/deep-dives/reactiveui/data-persistence
